verification/curator-ui/pages/index.js

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head'
+
+export default function Home() {
+  return (
+    <div className="container">
+      <Head>
+        <title>Curator UI</title>
+      </Head>
+
+      <main>
+        <h1>Welcome to the Curator UI</h1>
+      </main>
+
+      <style jsx>{`
+        .container {
+          min-height: 100vh;
```

Comment:
Super-nit: I usually alphabetize CSS properties because it makes it easy to search/parse/find duplicates.

Reply:
Done